routing_common/vehicle_model.hpp

```diff
@@ -54,7 +54,7 @@ enum class HighwayType : uint16_t
   HighwaySecondaryLink = 176,
   RouteFerry = 259,
   HighwayTertiaryLink = 272,
-  HighwayBusway = 858,    // reserve type here, but this type is not used for any routing by default
+  HighwayBusway = 857,    // reserve type here, but this type is not used for any routing by default
```

Comment:
Why? I don't see changes in types.txt here.

Can we make a test for these constants, not to break something in future? :)

Reply:
Its just wrong, it should be a mapping ID - 1, so atm it points to highway-busway-bridge
```
highway|busway;[highway=busway],[highway=service][service=busway],[highway=service][service=bus];;name;int_name;858;
highway|busway|bridge;[highway=busway][bridge?];;name;int_name;859;
```